libs/cardano-ledger-core/src/Cardano/Ledger/Core/Era.hs

```diff
@@ -20,7 +22,8 @@ module Cardano.Ledger.Core.Era (
   -- ** Rules
   EraRule,
   EraRuleFailure,
-  VoidRuleFailure,
```

Comment:
Should we mention the removal of `VoidRuleFailure` in CHANGELOG ?

Reply:
Yes, `VoidEraRule` is a replacement. I'll adjust the changelog. thanks for noticing